index.js

```diff
@@ -249,7 +255,16 @@ class ServerlessCustomDomain {
       throw new Error(`${action} is not a valid action. action must be either CREATE or DELETE`);
     }
 
+    if (this.serverless.service.custom.customDomain.createRoute53Record !== undefined
```

Comment:
Is the `this.serverless.service.custom.customDomain.createRoute53Record !== undefined` check needed if its also doing `this.serverless.service.custom.customDomain.createRoute53Record === false`?